Xamarin/SSW.Rewards/SSW.Rewards.Android/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="1.78" package="com.ssw.consulting" android:installLocation="auto" android:versionCode="17">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="1.79" package="com.ssw.consulting" android:installLocation="auto" android:versionCode="18">
```

Comment:
Can we keep this at 1.78 for now? It should be fine to change the version code but not the name. We want to keep these consistent across iOS and Android.

Reply:
np, done :)